data/events/2024/kerala/main.yml

```
@@ -37,9 +37,11 @@ location_address: "Le Méridien Kochi, Nettoor, Maradu, Kochi, Ernakulam, Kerala
 nav_elements: # List of pages you want to show up in the navigation of your page.
  # - name: propose
  # - name: location
- # - name: registration
+   - name: register
+     url: https://konfhub.com/devopsdays-kerala-2024
```

Whoops, I had an extra set of spaces. That's why the preview doesn't show anything. If you can accept these changes (or update and commit yourself), I can merge. (p.s., if you want to check this locally, are comfortable doing so, and you already have hugo installed, a `hugo serve` works just fine.)
```suggestion
  - name: register
    url: https://konfhub.com/devopsdays-kerala-2024
```

---

Thanks.. I missed it too.. Have accomodated the changes